experimenter/experimenter/nimbus_ui_new/filtersets.py

```diff
@@ -302,8 +303,12 @@ def filter_type(self, queryset, name, value):
 
     def filter_takeaways(self, queryset, name, value):
         query = Q()
-        if NimbusExperiment.Takeaways.DAU_GAIN in value:
-            query |= Q(takeaways_metric_gain=True)
-        if NimbusExperiment.Takeaways.QBR_LEARNING in value:
-            query |= Q(takeaways_qbr_learning=True)
+        for val in value:
+            if val in dict(NimbusExperiment.Takeaways.choices):
```

Comment:
do we really need to call dict() here? I'm sure theres something available on Takeaways